index.js

```diff
@@ -1,3 +1,21 @@
+const percision = 4;
+const lhc = require('./libs/LighthouseConvert');
+
 exports.hello = function () {
   console.log("This is a message from the lighthouse-unit-conversion package");
 };
+
+exports.setDecimalPlaces = (newPercision) => {
+  if (isNaN(newPercision)) return;
+  percision = newPercision;
+}
+
+exports.celsiusToFahren = (celsius) => {
```

Comment:
celsiusToFahrenheir. Btw I kinda want the precision to be in the second params and default it to a value (4?)

Reply:
Switched to toFixed instead of precision